tutorials/ner_pytorch_medical/project.yml

```diff
@@ -0,0 +1,109 @@
+title: "Detecting entities in Medical Records with PyTorch"
+description: "This project uses the [i2b2 (n2c2) 2011 Challenge Dataset](https://portal.dbmi.hms.harvard.edu/projects/n2c2-nlp/) to bootstrap a PyTorch NER model to detect entities in Medical Records. It also demonstrates how to anonymize medical records for annotators in [Prodigy](https://prodi.gy)."
+# Variables can be referenced across the project.yml using ${vars.var_name}
+vars:
+  spacy_config: "spacy_config.cfg"
+  config: "config.cfg"
+  config_trf: "config_trf.cfg"
+  name: "ner_pytorch_medical"
+  version: "0.0.0"
+  train: "train"
+  dev: "dev"
+  test: "test"
+
+# These are the directories that the project needs. The project CLI will make
+# sure that they always exist.
+directories: ["assets", "training", "configs", "scripts", "corpus", "packages"]
+
+# Workflows are sequences of commands (see below) executed in order. You can
+# run them via "spacy project run [workflow]". If a commands's inputs/outputs
+# haven't changed, it won't be re-run.
+workflows:
+  all:
+    - preprocess
+    - train
+    - evaluate
+
+# Project commands, specified in a style similar to CI config files (e.g. Azure
+# pipelines). The name is the command name that lets you trigger the command
+# via "spacy project run [command] [path]". The help message is optional and
+# shown when executing "spacy project run [optional command] [path] --help".
+commands:
+  - name: "preprocess"
+    help: "Convert the data to spaCy's binary format"
+    script:
+      - "python scripts/preprocess.py assets/n2c2_2011 corpus"
+    deps:
+      - "assets/n2c2_2011/i2b2_Beth_Train_Release.tar.gz"
+      - "assets/n2c2_2011/i2b2_Partners_Train_Release.tar.gz"
+      - "assets/n2c2_2011/Task_1C.zip"
+    outputs:
+      - "corpus/${vars.train}.spacy"
+      - "corpus/${vars.dev}.spacy"
+      - "corpus/${vars.test}.spacy"
+
+  - name: "train"
+    help: "Train a custom PyTorch named entity recognition model"
+    script:
+      - "python -m spacy train configs/${vars.config} --output training/ --paths.train corpus/${vars.train}.spacy --paths.dev corpus/${vars.dev}.spacy --code scripts/custom_functions.py --components.tok2vec.model.embed.include_static_vectors true --paths.vectors en_core_web_lg --gpu-id 0"
```

Comment:
Is there a reason why you're having the config overrides here, and not in the config?

Reply:
Not at all haha, that's a good point. I'll guess I'll just remove this? I feel like assuming the en_core_web_lg vectors are downloaded is likely bad practice for the config files but I'm not sure.

Reply:
You could also have a `spacy download` command as part of the project workflow, then the assumption is explicit :-)